doc/iterations/2024.5/shapes/guarantee_representations_models_compatibility_between_versions.adoc

```diff
@@ -0,0 +1,39 @@
+= Add the support for representations migration
+
+== Context
+
+Users want to upgrade the version of Sirius Web without losing the representatons they created with an older version.
+
+== Key Result
+
+When a user creates a representation with a version n of Sirius Web, the representation created must be available with any version>n of Sirius Web. 
+
+The user must also be able to contribute on how to handle evolutions on his own representation to guarantee compatibility between several versions.
+
+== Solution - Sirus Web Representations
+
+1. Each representation must be identified by a version number.
```

Comment:
We will not "identify" representation by a version number. They will have a version number but it will not be used as an identifier. I won't look for a representation by giving its versions number.